docs/FURPS/communities-over-waku.md

```diff
@@ -0,0 +1,51 @@
+# Waku Requirements for Status Communities — FURPS
+
+Note: These requirements assume the Community Description has been moved to Codex
+
+## Functionality
+
+- **Community messages** (channel posts, updates) must be sent and received via **Waku**.
```

Comment:
This requirement makes sense, but out of curiosity is Waku a hard requirement here?